Gemfile.lock

```diff
@@ -1,61 +1,65 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (6.0.3.4)
-      actionpack (= 6.0.3.4)
+    actioncable (6.1.3.1)
```

Comment:
These get updated when you unlock the dependency graph and update the gds-sso gem